frontend/src/components/AppSidebar.tsx

```diff
@@ -0,0 +1,401 @@
+import React from "react";
+
+import { Link, NavLink, useNavigate } from "react-router-dom";
+import ExternalLink from "src/components/ExternalLink";
+import { AlbyHubIcon } from "src/components/icons/AlbyHubIcon";
+import { AlbyHubLogo } from "src/components/icons/AlbyHubLogo";
+import SidebarHint from "src/components/SidebarHint";
+import {
+  DropdownMenu,
+  DropdownMenuContent,
+  DropdownMenuGroup,
+  DropdownMenuItem,
+  DropdownMenuLabel,
+  DropdownMenuSeparator,
+  DropdownMenuTrigger,
+} from "src/components/ui/dropdown-menu";
+import {
+  Sidebar,
+  SidebarContent,
+  SidebarFooter,
+  SidebarGroup,
+  SidebarGroupContent,
+  SidebarHeader,
+  SidebarMenu,
+  SidebarMenuButton,
+  SidebarMenuItem,
+  useSidebar,
+} from "src/components/ui/sidebar";
+import {
+  Tooltip,
+  TooltipContent,
+  TooltipProvider,
+  TooltipTrigger,
+} from "src/components/ui/tooltip";
+import { UpgradeDialog } from "src/components/UpgradeDialog";
+import UserAvatar from "src/components/UserAvatar";
+import { useAlbyInfo } from "src/hooks/useAlbyInfo";
+import { useAlbyMe } from "src/hooks/useAlbyMe";
+import { useHealthCheck } from "src/hooks/useHealthCheck";
+import { useInfo } from "src/hooks/useInfo";
+import { deleteAuthToken } from "src/lib/auth";
+import { cn } from "src/lib/utils";
+
+import { isHttpMode } from "src/utils/isHttpMode";
+
+export function AppSidebar() {
+  const { data: albyMe } = useAlbyMe();
+
+  const { data: info, mutate: refetchInfo } = useInfo();
+  const { isMobile, state } = useSidebar();
+  const { hasChannelManagement } = useInfo();
+  const navigate = useNavigate();
+
+  const _isHttpMode = isHttpMode();
+
+  const logout = React.useCallback(async () => {
+    deleteAuthToken();
+    await refetchInfo();
+
+    if (_isHttpMode) {
+      window.location.href = "/logout";
+    } else {
+      navigate("/", { replace: true });
+    }
+  }, [_isHttpMode, navigate, refetchInfo]);
+
+  const data = {
+    user: {
+      name: "shadcn",
+      email: "m@example.com",
+      avatar: "/avatars/shadcn.jpg",
+    },
+    navMain: [
+      {
+        title: "Home",
+        url: "/home",
+        icon: HomeIcon,
+      },
+      {
+        title: "Wallet",
+        url: "/wallet",
+        icon: WalletIcon,
+      },
+      {
+        title: "App Store",
+        url: "/appstore",
+        icon: LayoutGridIcon,
+      },
+      {
+        title: "Connections",
+        url: "/apps",
+        icon: Plug2Icon,
+      },
+    ],
+    navSecondary: [
+      ...(hasChannelManagement
+        ? [
+            {
+              title: "Node",
+              url: "/channels",
+              icon: BoxIcon,
+            },
+          ]
+        : []),
+      {
+        title: "Settings",
+        url: "/settings",
+        icon: Settings,
+      },
+    ],
+  };
+
+  return (
+    <Sidebar collapsible="icon">
+      <SidebarHeader>
+        {state === "expanded" ? (
+          <div className="p-2 flex flex-row items-center justify-between">
```

Comment:
In mobile we see a lot of space between the shield and indicator, but not sure if this is even worth it since we are going to remove it in near future

<img width="211" alt="Screenshot 2025-03-25 at 2 10 05 PM" src="https://github.com/user-attachments/assets/d7023958-c042-4bae-906e-068b900f8e74" />

Reply:
we can right-align the 2 items (shield + indicator) instead of spacing them out :+1: